modules/sievefilters/functions.php

```diff
@@ -361,9 +361,9 @@ function block_filter($filter, $user_config, $action, $imap_server_id, $sender,
 }
 
 if (!hm_exists('block_filter_dropdown')) {
-    function block_filter_dropdown($mod, $with_scope = true, $submit_id = 'block_sender', $submit_title = 'Block') {
-        $ret = '<div class="dropdown-menu p-3" id="dropdownMenuBlockSender">'
-            .'<form id="block_sender_form">';
+    function block_filter_dropdown ($mod, $mailbox_id, $with_scope = true, $submit_id = 'block_sender', $submit_title = 'Block', $increment = "") {
```

Comment:
`$mailbox_id` should have default value because somewhere else like here: https://github.com/christer77/cypht/blob/Cypht-edit-a-block/modules/imap/output_modules.php#L390 we could have error as it's not passed and doesn't have default value.

Reply:
Before making this change, I had reassured myself if there is another call to the function than in this file, fortunately not.

Reply:
https://github.com/christer77/cypht/blob/Cypht-edit-a-block/modules/imap/output_modules.php#L390

Reply:
I see